source/utilities.tex

```diff
@@ -9653,6 +9714,10 @@
   shared_ptr<T> make_shared(@\placeholdernc{args}@);
 template<class T, class A, ...>
   shared_ptr<T> allocate_shared(const A& a, @\placeholdernc{args}@);
+template<class T, ...>
+  shared_ptr<T> make_shared_default_init(@\placeholdernc{args}@);
+template<class T, class A, ...>
+  shared_ptr<T> allocate_shared_default_init(const A& a, @\placeholdernc{args}@);
```

Comment:
This seems broken (also in the ingress paper): The whole point of ..._default_init is to have no arguments, so why are we talking about "args" here? Also, the pack-like "..." in the template argument list should go.

Reply:
Ah, that's to cover "size_t N" for the array case. The not-really parallelism to make_shared(args) is highly misleading.